Library/Formula/cpp-netlib.rb

```diff
@@ -27,7 +35,8 @@ class CppNetlib < Formula
   def install
     ENV.cxx11
 
-    system "cmake", *std_cmake_args
+    # NP: Do not build examples or tests as they require submodules.
```

Comment:
Should this be "NB" ("nota bene") instead of "NP" ("nurse practitioner"/"no problem"/"non-polynomial")?